README.md

```diff
@@ -46,6 +46,23 @@ Unit tests can be running using the following `npm` command:
 ```bash
 npm run test
 ```
+
+## Usage
+
+FQM-Testify is a React-based web application for analyzing FHIR Electronic Clinical Quality Measures (eCQMs) through the creation of test FHIR Patients and resources.
+
+### Creating a Test Case
+
+For instructions on starting the app, check [here](README.md#L20). After starting up the app, begin by uploading a FHIR Measure Bundle from your local machine. The measurement period start and end pickers will update to reflect the effectivePeriod of the Measure resource in the uploaded bundle.
```

Comment:
The README link isn't working for me :/ not sure if it's because this hasn't been merged yet. I'm not sure if we necessarily need to redirect to the local installation instructions anyway since they aren't too far away from the Usage section.